new/LIT6751PPAynat.xml

```diff
@@ -0,0 +1,80 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6751PPAynat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer against the evil eye (<foreign xml:lang="gez">ʿāynat</foreign>),
```

Comment:
Etymologically ʿāynat is ʿaynat (Cp, Leslau, Comp. Dict. p. 80a), unless a different spelling form has been established and adopted in BM

Reply:
You're right (and it was never discussed), but in my estimation the spelling with ዓ is the pervasive one, so I would tendentially be against a normalisation in this case.
Cp. https://betamasaheft.eu/as.html?query=%E1%8B%93%E1%8B%AD%E1%8A%90%E1%89%B5&AttestedInType=1&xmlid= and https://betamasaheft.eu/as.html?query=%E1%8B%90%E1%8B%AD%E1%8A%90%E1%89%B5&AttestedInType=1&xmlid=

Reply:
ዓይነት፡ is indeed more frequent, even though the etymological spelling form remains ዐይነት. However, let's stick to the popular form. We might discuss and standardize these forms at a later stage

Reply:
It's an interesting discussion! I believe that there might be a diachronic dimension to many of these cases as well, but it remains to be studied more in depth.